R/deps.R

```diff
@@ -407,7 +429,13 @@ standardise_dep <- function(x) {
   } else if (identical(x, FALSE)) {
     character(0)
   } else if (is.character(x)) {
-    x
+    if (any(x %in% c("hard", "user"))) {
+      c("Depends", "Imports", "LinkingTo")
+    } else if (any(x %in% c("soft", "dev"))) {
```

Comment:
Well then you would just pass `Config/Depends/devel` in that case...

Reply:
But it would make sense to include `Config/Depends/devel` in `dev` by default. If you don't include it now, then you'll break the API when you add it later.

Reply:
Ok, I got rid of the user / dev options